pkg/sypgp/sypgp.go

```diff
@@ -71,28 +72,35 @@ func AskQuestionNoEcho(format string, a ...interface{}) (string, error) {
 	return string(response), nil
 }
 
-// GetTokenFile returns a string describing the path to the stored token file
-func GetTokenFile() string {
+// getSingularityDir returns the directory where the user's singularity
+// configuration and data is located.
+func getSingularityDir() string {
 	user, err := user.GetPwUID(uint32(os.Getuid()))
 	if err != nil {
-		sylog.Warningf("could not lookup user's real home folder %s", err)
-		sylog.Warningf("using current directory for %s", filepath.Join(".singularity", "sylabs-token"))
-		return filepath.Join(".singularity", "sylabs-token")
+		sylog.Warningf("could not lookup user's real home directory: %s", err)
```

Comment:
I'm fine with this warning starting with a non-cap, but to be consistent with the rest of the code, it might be better to start this with a cap:

```suggestion
		sylog.Warningf("Could not lookup user's real home directory: %s", err)
```